js/components/badges/modal.vue

```diff
@@ -63,31 +66,21 @@ export default {
         },
         hasModifications() {
             return (this.selected.length !== this.initial.length)
-                || this.selected.some((badge) => {
-                    return this.initial.indexOf(badge) < 0;
-                });
+                || this.selected.some(badge => this.initial.indexOf(badge) < 0);
         }
     },
     compiled() {
         this.badges = badges.available(this.subject);
 
         if (this.subject.hasOwnProperty('badges')) {
-            this.selected = this.subject.badges.map(function(badge) {
-                return badge.kind;
-            });
-
+            this.selected = this.subject.badges.map(badge => badge.kind);
             this.initial = this.selected.slice(0);
         }
     },
     methods: {
         confirm() {
-            const to_add = this.selected.filter((badge) => {
-                        return this.initial.indexOf(badge) < 0;
-                    }),
-                to_remove = this.initial.filter((badge) => {
-                        return this.selected.indexOf(badge) < 0;
-                    });
-
+            const to_add = this.selected.filter(badge => this.initial.indexOf(badge) < 0);
```

Comment:
As much as possible, camel case in JS. As this PR is already a big one I tried to limit changes and so snake_case is kept where it is already.